modules/cms-kit/src/Volo.CmsKit.MongoDB/Volo/CmsKit/MongoDB/Blogs/MongoBlogPostRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Threading.Tasks;
+using Volo.Abp.Domain.Repositories.MongoDB;
+using Volo.Abp.MongoDB;
+using Volo.CmsKit.Blogs;
+
+namespace Volo.CmsKit.MongoDB.Blogs
+{
+    public class MongoBlogPostRepository : MongoDbRepository<CmsKitMongoDbContext, BlogPost, Guid>, IBlogPostRepository
+    {
+        public MongoBlogPostRepository(IMongoDbContextProvider<CmsKitMongoDbContext> dbContextProvider) : base(dbContextProvider)
+        {
+        }
+
+        public Task<BlogPost> GetByUrlSlugAsync(string urlSlug)
+        {
+            return GetAsync(x => x.UrlSlug.ToLower() == urlSlug);
+        }
+
+        public async Task<bool> SlugExistsAsync(string slug)
+        {
+            var queryable = await GetQueryableAsync();
+
+            return await AsyncExecuter.AnyAsync(queryable, x => x.UrlSlug.ToLower() == slug);
```

Comment:
Is there any special case to use `AsyncExecuter`?

Reply:
`AnyAsync` doesn't exist on **IMongoQueryable**, if you have a better solution, I'm always open for bbetter suggestions @cotur